lib/version_helper.rb

```diff
@@ -7,7 +7,7 @@ def initialize(command)
 
   def check_version
     color "Checking for zappifest update...", :green
-    @latest_zappifest_stable = `brew info zappifest`.split("\n")[0].split(" ")[-1]
+    @latest_zappifest_stable = `curl https://raw.githubusercontent.com/applicaster/homebrew-tap/master/zappifest.rb | sed -n 5p `.split(" ")[-1].delete('"')
```

Comment:
I feel like it would be better to get the version by `grep`, which would be more stable. Something like
`grep "version \"(\d\+\.\d\+\.\d\+)\""`
Although probably better to use `ruby`'s regex system...

Reply:
yeah you're right. Actually I don't have to split it by line - if for some reason the version is non longer on line 5, it will break.

I can do `result.scan(/version \".+\"/)` to extract `version "xx.xx.xx-yy.zz"` and parse the version from there 👍